types/src/on_chain_config/aptos_features.rs

```diff
@@ -47,6 +47,20 @@ impl Features {
         byte_index < self.features.len() && (self.features[byte_index] & bit_mask != 0)
     }
 
+    pub fn update(&mut self, flag: FeatureFlag, enabled: bool) {
+        let bit_idx = flag as usize;
+        let byte_idx = bit_idx / 8;
+        if self.features.len() < byte_idx + 1 {
+            self.features.resize(byte_idx + 1, 0);
+        }
+        let mask = 1_u8 << (bit_idx % 8);
+        if enabled {
+            self.features[byte_idx] |= mask;
+        } else {
+            self.features[byte_idx] &= !mask;
+        }
+    }
+
```

Comment:
this isn't tested, do we really need it?

Reply:
if so, please add some tests

Reply:
updated the other place so this method is no longer needed.